src/views/product-integration/index.tsx

```diff
@@ -69,54 +74,70 @@ export default function ProductIntegrationLanding({
 		})
 	}
 
+	// Generate Sidebar Information
+	const currentProduct = useCurrentProduct()
+	const sidebarNavDataLevels = [
+		generateTopLevelSidebarNavData(currentProduct.name),
+		generateProductLandingSidebarNavData(currentProduct),
+		{
+			backToLinkProps: {
+				text: `${currentProduct.name} Integrations Home`,
+				href: `/${currentProduct.slug}/integrations`,
+			},
+			levelButtonProps: {
+				levelUpButtonText: `${currentProduct.name} Home`,
+				levelDownButtonText: 'Previous',
+			},
+			menuItems: [],
+			showFilterInput: false,
+			title: integration.name,
+		},
+	]
+
 	return (
-		<BaseLayout showFooterTopBorder>
-			{notLatest && (
-				<HashiHead>
-					<meta name="robots" content="noindex, nofollow" />
-				</HashiHead>
-			)}
-			<div className={s.integrationPage}>
-				<div className={s.sidebar}></div>
-				<div className={s.mainArea}>
-					<div className={s.contentWrapper}>
-						<div className={s.breadcrumbWrapper}>
-							<BreadcrumbBar links={breadcrumbLinks} />
-						</div>
-						<div className={s.content}>
-							<Header
-								className={s.header}
-								integration={integration}
-								activeRelease={activeRelease}
-								versions={versions}
-								onInstallClicked={() => {
-									// TODO: we'll need to flip the tab over to the install tab here
-									console.log('TODO: Install Clicked')
-								}}
-							/>
-							<Tabs
-								allowNestedStyles
-								// translate slug to index
-								initialActiveIndex={map.slugToIndex[tab]}
-								onChange={handleTabChange}
-							>
-								<Tab heading="README">
-									<ReactMarkdown>{activeRelease.readme}</ReactMarkdown>
+		<SidebarSidecarLayout
+			sidebarNavDataLevels={sidebarNavDataLevels}
+			breadcrumbLinks={breadcrumbLinks}
```

Comment:
TIL: `breadcrumbLinks` prop — I'm going to copy this for `product-integrations-landing`

Reply:
Indeed I was like ✨ when I discovered this yesterday myself.